recipe/0001-fix-attrs-dropping.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/xcdat/temporal.py b/xcdat/temporal.py
+index 3120d48..2bb35e6 100644
+--- a/xcdat/temporal.py
++++ b/xcdat/temporal.py
+@@ -1147,7 +1147,7 @@ class TemporalAccessor:
+             # dimension) shape of the `weights` DataArray to the
+             # multi-dimensional shape of its corresponding data variable.
+             weights, _ = xr.broadcast(self._weights, dv)
+-            weights = xr.where(np.isnan(dv), 0.0, weights)
++            weights = xr.where(dv.copy().isnull(), 0.0, weights)
```

Comment:
In `xarray>=2023.03.0`, `np.isnan()` drops the attributes of `dv` (an `xarray.DataArray`) for some unknown reason.

The solution is to make a copy `dv` and call xarray's `isnull()` on the object.